README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status]][github-ci] [![Coverage Status]][coveralls.io]
 [![Latest Version]][crates.io] [![docs]][docs.rs]
-[![Minimum Supported Rust Version]][Rust 1.52]
+[![Minimum Supported Rust Version]][Rust 1.57]
```

Comment:
`union` casts in `const fn`s were stabilized in Rust 1.56, so as a follow up to this PR many function could become `const fn`. cc #76

Reply:
Oh, interesting. Thanks for the heads up.